docs/docs/20-usage/60-services.md

```diff
@@ -77,3 +79,21 @@ services:
   database:
     image: mysql
 ```
+
+## Example
+
+```yml
+services:
+  database:
+    image: mysql
+    environment:
+      - MYSQL_DATABASE=test
+      - MYSQL_ROOT_PASSWORD=example
+pipeline:
+  get-version:
+    image: ubuntu
+    commands:
+      - ( apt update && apt dist-upgrade -y && apt install -y mysql-client 2>&1 )> /dev/null
+      - sleep 30s # need to wait for mysql-server init
+      - echo 'SHOW VARIABLES LIKE "version"' | mysql -u root -h database test -pexample
```

Comment:
```suggestion
      - echo 'SHOW VARIABLES LIKE "version"' | mysql -uroot -hdatabase test -pexample
```
or
```suggestion
      - echo 'SHOW VARIABLES LIKE "version"' | mysql -u root -h database test -p example
```
?

Reply:
Are both of those valid?
As far as I know, the `-p` flag is special in mysql in that it requires no space.

Reply:
AB testing:
 - https://ci.codeberg.org/6543/ci-service-example/build/11/4
 - https://ci.codeberg.org/6543/ci-service-example/build/13/4